recipes/default.rb

```diff
@@ -5,6 +5,14 @@
 
 agent_password = ""
 
+fqdn = node['fqdn']
+hostname = node['hostname']
```

Comment:
If you remove it from the `keystore.sh.erb`, then you can remove it from here as well.

Reply:
Ok, the hostname was no longer necessary. So i removed that. 
I also removed the chef attribute:
kagent/hostname
Nobody is using it, and it's too complex to support. IMO.